docker-entrypoint.sh

```diff
@@ -29,6 +29,16 @@ export COMPOSER_HOME="/tmp"
 # Provide simple fingerprint for detecting that a deployment is in progress
 export CLOUD_BUILD=1
 
+if [ -f "/tmp/hosts.ext" ]; then
```

Comment:
What is the purpose of adding a mount point for `/etc/hosts` additions in the build box? Is there a domain we want to enforce to IP somewhere?

Reply:
Yes, this is how a lot of things on CWP work - there is currently overrides on dash, so that the ssh traffic doesn't go out and back in, but instead goes straight to gitlab. This is what John recommended.

Reply:
If its predefined hosts (e.g. Gitlab IP address) I think using the `docker run --add-hosts` functionality would be better suited.
https://docs.docker.com/engine/reference/run/#network-settings

E.g.
```docker run ... --add-host gitlab.cwp.govt.nz:<ip-address>``` ...

This should add the host and ip to the `/etc/hosts` file without having to use volume mounts and custom bash scripts.

Reply:
ooooh nice will do

Reply:
Done